.github/workflows/unit-tests.yml

```
@@ -46,3 +65,37 @@ jobs:
         with:
           name: ${{github.run_number}}-${{github.run_attempt}}-coverage
           path: coverage-artifacts/**/*
+  unit-test-sbomscanner:
+    if: github.repository_owner == 'rancher' && ${{ needs.detect-changes.outputs.sbomscanner == 'true' }}
+    needs: detect-changes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+
+      - name: Setup yarn
+        run: npm install -g yarn
+
+      - name: Setup Nodejs with yarn caching
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run tests and collect coverage
```

We have shared `unit-tests.yml` file for both extensions, but the only shared code is `on` scheduling block.

Maybe I missed something, but could we share all the setup code and only trigger specific test?
I mean we can have only 1 job, all the code would be shared with something like:

```suggestion
      - if: ${{ needs.detect-changes.outputs.kubewarden == 'true' }}
        run: |
          npm run test:ci:kubewarden
          ...
      - if: ${{ needs.detect-changes.outputs.sbomscanner == 'true' }}
        run: |
          npm run test:ci:sbomscanner
          ...
```